method/STAGATE/method_STAGATE.py

```diff
@@ -119,12 +119,24 @@ def get_anndata(args):
 np.random.seed(seed)
 
 import STAGATE as sg
-# Construct the spatial network
-if config["model"] == 'Radius':
+from scipy.spatial.distance import euclidean
+# To skip the neighbor computation step in STAGATE
+if args.neighbors is not None:
+    neighbors = sp.io.mmread(args.neighbors).T.tocsr()
+    neighbors = neighbors.toarray()
+    row_indices, col_indices = np.where(neighbors == 1)
```

Comment:
Converting to dense is not a good idea.
Instead you can do
`neighbors = sp.io.mmread(args.neighbors).T.tocoo()`
`row_indices, col_indices = neighbors.row, neighbors.col`